File: backend/src/main/java/cohappy/backend/model/dto/CreateHouseDTO.java
package cohappy.backend.model.dto;

import lombok.Data;

import java.util.List;

@Data
public class CreateHouseDTO {
    private String userCode;
    private List<byte[]> images;
    private Integer costPerMonth;
    private String country;
    private String region;
    private String street;
    private Integer civicNumber;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/request/CreateHouseDTO.kt
package cohappy.frontend.model.dto.request
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class CreateHouseDTO(
    val userCode: String,
    var images: List<ByteArray>? = null,
    val costPerMonth: Int, //INTEGER
    val country: String,
    val region: String,
    val street: String,
    val civicNumber: Int //INTEGER
)
